cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/utils/StorageUtils.java

```diff
@@ -92,4 +119,28 @@ public static List<FrameInterval> sliceInterval (Interval interval) {
 
         return slices;
     }
+
+    public static LocalDate toLocalDate(Instant instant) {
+        if (instant.isBefore(MIN_EPOCH_INSTANT)) {
+            LOGGER.trace("Replaces {} by {}", instant, MIN_EPOCH_INSTANT);
+            return LocalDate.MIN;
```

Comment:
Does cassandra work fine with `LocalDate.MIN`? This will be the value like this `-999999999-01-01` according to Javadoc.

Reply:
Yes, this case is checked in the tests